lib/modules/dosomething/dosomething_campaign/dosomething_campaign.strongarm.inc

```diff
@@ -15,7 +15,7 @@ function dosomething_campaign_strongarm() {
   $strongarm->api_version = 1;
   $strongarm->name = 'entity_translation_settings_node__campaign';
   $strongarm->value = array(
-    'default_language' => 'xx-et-default',
+    'default_language' => 'xx-et-author',
```

Comment:
@deadlybutter we want to keep this as `default` (it's a setting in that entity translate tab) 
you can just make the change to the file manually, and amend the commit, or export the setting again
http://superuser.com/questions/229290/how-to-amend-the-last-commit-to-un-add-a-file